### PR TITLE
Moved the whole widget code to OpenRA.Mods.Common

### PR DESCRIPTION
**Pull request by Mailaender**
I need this now for #4496.

### CODE REVIEW

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/DebugMenuLogic.cs [hunk 1]
```diff
@@ -10,7 +10,6 @@
 
 using System;
 using OpenRA.Mods.Common.Traits;
-using OpenRA.Mods.RA.Traits;
 using OpenRA.Support;
 using OpenRA.Traits;
 using OpenRA.Widgets;
```
Comment (penev92) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/DebugMenuLogic.cs]: You missed to change this class's namespace.

**Comment by penev92** [comment]
Otherwise looks OK to me :+1:

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/DebugMenuLogic.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/DebugMenuLogic.cs]: Fixed.

**Comment by obrakmann** [comment]
To me, too. :+1:

**Pull request merged**